docs/tutorial/create-entity.md

```diff
@@ -11,49 +11,76 @@ Entities have properties (fields), actions (methods) and often is uniquely ident
 
 > Refer to [Getting Started - What's and Entity](/docs/entity/getting-started#whats-an-entity) to know more.
 
-The most important entity is the User, so we are going to create it.
+The most basic entity is the User. The CLI generates it out-of-the-box, so let's understand how it works:
 
-## List Entity
+## User Entity
 
 ### Entity name
 
-First, let's set the name for the entity:
+First, let's understand how is set the name for the entity:
 
 ```js
 // src/domain/entities/user.js
 const { entity, field } = require('@herbsjs/herbs')
 
+// The entity name is first argument of `entity()`
 const User = entity('User', {})
 ```
 
 ### Entity fields
 
-Now, we're going to set the fields for the List entity:
+Now, we're going to see the fields for the User entity:
+
+- id: The unic identificator for the user.
+- nickname: The nickname for the user like "user123".
+- password: The user's acess password.
 
 ```js
 // src/domain/entities/user.js
 const { entity, field } = require('@herbsjs/herbs')
 
+// The second argument is an object
+// with the fields.
 const User = entity('User', {
+    // The key is the field name.
+    // The value is the object type of
+    // the field using the `field()`.
+
+    // The field "id" is a number.
     id: field(Number),
+
+    // Both the fields "nickname" and
+    // "password" are texts, therefore
+    // we are using `String`.
```

Comment:
Maybe put a section about scalar types like: For scalar types a default value is assumed if a default value is not given:

https://github.com/herbsjs/gotu/blob/master/README.md